inc/class-wpseo-replace-vars.php

```diff
@@ -1146,15 +1146,85 @@ private function get_replacement_variables() {
 	 *
 	 * @param string $replacement_variable The replacement variable.
 	 *
-	 * @return bool True when the replacement variable contains a
+	 * @return bool True when the replacement variable is not prefixed.
 	 */
 	private function is_not_prefixed( $replacement_variable ) {
-		$prefixes = array( 'cf_', 'ct_' );
-		$prefix   = substr( $replacement_variable, 0, 3 );
+		$prefixes = array( 'cf_', 'ct_', 'pt_' );
+		$prefix   = $this->get_prefix( $replacement_variable );
 
 		return ! in_array( $prefix, $prefixes );
 	}
 
+	/**
+	 * Strip the prefix from a replacement variable name.
+	 *
+	 * @param string $replacement_variable The replacement variable.
+	 *
+	 * @return string The replacement variable name without the prefix.
+	 */
+	private function strip_prefix( $replacement_variable ) {
+		return substr( $replacement_variable, 3 );
```

Comment:
Keep in mind that in a future the prefixes can be longer... This is something we should fix later, but we shouldn't forget it is possible.

Reply:
But this is only done for the replace_vars in the help_texts. The idea (I think) was to strip the `ct_<custom-field-name>` etc. rows from the array. So IF we implement prefixes that are longer to the replace vars (and thus the help texts), then we should change this. But we have full control, so ( I expect :trollface: ) no unexpected behaviour.